### ISSUE TITLE
NTM Newsgroup example code uses wrong parameter name for validation data

### ISSUE THREAD

**Issue by bt-jjh** [aws/amazon-sagemaker-examples#440]
This example notebook

https://github.com/awslabs/amazon-sagemaker-examples/blob/master/introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb

has this line to train the NTM model

ntm.fit({'train': s3_train, 'test': s3_val_data})

'test' is the incorrect parameter for validation dataset; it should be 'validation'

**Comment by bt-jjh** [comment]
These comments from the notebook indicate early stopping is intended:

> As a common practice in modeling training, we should have a training set, a validation set, and a test set. The training set is the set of data the model is actually being trained on. But what we really care about is not the model's performance on training set but its performance on future, unseen data. Therefore, during training, we periodically calculate scores (or losses) on the validation set to validate the performance of the model on unseen data. By assessing the model's ability to generalize **we can stop the training at the optimal point via early stopping to avoid over-training.**

**Comment by hsl89** [comment]
Dear SageMaker Community,
             In our attempts to provide this repository with a better level of support             going forward, we’re closing issues that were opened prior to the v2 release of the SDK.             This is because we believe that over time many of the issues             posted were solved with the latest release or other recent changes to the repo.             This will help us reallocate resources towards issues that are            more likely to still be relevant today.            Some of the issues experienced now can be resolved by referencing the            v2 guide: https://sagemaker.readthedocs.io/en/stable/v2.html             In this guide, you can find simple solutions to common notebook errors,             like the renaming of parameters and classes.            If you believe your issue is still ongoing and you have updated error            messaging or other info, please re-open it, and we investigate the issue.
            Best Regards,
            AWS SageMaker Team
